react/RatingSummary.tsx

```diff
@@ -122,6 +123,18 @@ const RatingSummary: FunctionComponent<Props> = props => {
         <Fragment>Loading reviews...</Fragment>
       ) : state.total === 0 ? null : (
         <Fragment>
+          <Helmet>
+            <script type="application/ld+json">{`{
+              '@context': 'http://schema.org',
```

Comment:
I think it might be easier to do something like this:

https://github.com/vtex-apps/structured-data/blob/master/react/Product.js#L140

So the linter won't complain. I'm afraid those trailing commas will break the JSON parser.

Reply:
Hmm, I was using dangerouslySet earlier, and Helmet was skipping over the scripts. I'm not sure why. I can remove the commas, in any case.

Reply:
Removed trailing commas